src/Symfony/Component/Console/Tests/ApplicationTest.php

```diff
@@ -491,6 +491,10 @@ public function testRenderException()
 
     public function testRenderExceptionWithDoubleWidthCharacters()
     {
+        if (!function_exists('mb_strwidth')) {
```

Comment:
As a reminder that on mb_strwidth is required here (but this could be removed as mb_strwidth is broken anyway, see e.g. https://github.com/nicolas-grekas/Patchwork-UTF8/blob/master/class/Patchwork/Utf8.php#L602 for a working implementation of string width on unicode terminals.